app/models/order_detail.rb

```diff
@@ -753,7 +753,11 @@ def can_dispute?
   end
 
   def in_dispute?
-    dispute_at.present? && dispute_resolved_at.nil? && !canceled?
+    dispute_at.present? && dispute_resolved_at.nil? && !canceled? && !auto_disputed?
+  end
+
+  def auto_disputed?
+    account.auto_dispute_by.present?
```

Comment:
@jossim I'm actually thinking we may want to make this behavior a little less entwined with `auto_dispute_by`, and add a new over-rideable method to the `Account` class:
```suggestion
  def global_admin_dispute?
    account.global_admin_must_resolve_disputes?
```